data.md

```diff
@@ -420,8 +420,9 @@ A cons-list is used for the EVM wordstack.
     syntax WordStack ::= #drop ( Int , WordStack ) [function, functional]
  // ---------------------------------------------------------------------
     rule #drop(N, WS)         => WS                  requires notBool N >Int 0
-    rule #drop(N, .WordStack) => .WordStack
-    rule #drop(N, (W : WS))   => #drop(N -Int 1, WS) requires N >Int 0
+    rule #drop(N, WS)         => #drop(1, #drop(N -Int 1, WS)) requires N >Int 1
+    rule #drop(1, _ : WS)     => WS
+    rule #drop(1, .WordStack) => .WordStack
```

Comment:
This is no longer a total function, so cannot have `functional`, or you need to make it a total function.

Reply:
Um, I think it is still total.  But in any case, this has another issue, so I'll update it later.

Reply:
BTW, how to make this PR "draft"?

Reply:
Never mind. Now it is ready.